src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ProvideCorrectArgumentsToFormattingMethodsTests.cs

```diff
@@ -63,48 +137,106 @@ public class C
 {
     void Method()
     {
-        Console.Write("""", 1);
-        Console.Write(""{0}"", 1, 2);
-        Console.Write(""{0} {1}"", 1, 2, 3);
-        Console.Write(""{0} {1} {2}"", 1, 2, 3, 4);
-        Console.Write(""{0} {1} {2} {3}"", 1, 2, 3, 4, 5);
+        {|#0:" + invocation + @"(""{0} {1}"", 1)|};
+        {|#1:" + invocation + @"(""{0} {1} {2}"", 1)|};
+        {|#2:" + invocation + @"(""{0} {1} {2}"", 1, 2)|};
+
+        // These cases are not handled as we cannot find a parameter named format
```

Comment:
I was tempted to update the code to handle these cases but I was a bit worried about the perf impact of analyzing the extra overloads. Let me know what you think.

Reply:
I would go ahead and handle them.